### PR TITLE
[#1304] buttons in user component are now next to each other (fixes #1304)

### LINKED ISSUE #1304: Users table button alignment/cut off
Since the upgrade to 6, some of the buttons are running into one another and cut off:

image

### PR DESCRIPTION
**Pull request by i5o**

### CODE REVIEW

**Comment by lmmrssa** [comment]
@i5o It fixes layout broken but still it has buttons next to each other so can you add something for it.

**Comment by i5o** [comment]
Okay, so I added something to styles.scss I think this is what we need
image

image
image

File: src/styles.scss [hunk 1]
```diff
@@ -334,4 +334,8 @@ body {
   mat-dialog-actions {
     justify-content: flex-end;
   }
+
+  .mat-raised-button {
```
Comment (paulbert) [inline src/styles.scss]: This is overriding margins added via other classes to some buttons.  Let's just have this as part of the component style.

Comment (i5o) [reply src/styles.scss]: What do you mean by that @paulbert? 
Only adding it to the users component?

**Comment by i5o** [comment]
Hey @paulbert I moved the css to the component

**Comment by lmmrssa** [comment]
@paulbert @i5o can we have this style implemented all places or as global as all buttons are touching each other

**Pull request merged**